sentry_sdk/consts.py

```diff
@@ -79,6 +79,7 @@ class CompressionAlgo(Enum):
             ],
             "metric_code_locations": Optional[bool],
             "enable_logs": Optional[bool],
+            "before_send_log": Optional[Callable[[Log], Optional[Log]]],
```

Comment:
Thanks, I think it should be 
```suggestion
            "before_send_log": Optional[Callable[[Log, Hint], Optional[Log]]]
```
similar to `TransactionProcessor`

Reply:
Thanks, this is also wrong in the docs 🙈